epispot/models.py

```diff
@@ -3,40 +3,47 @@
 For example, the basic SIR model would be a model consisting of the Susceptible, Infected, and Recovered
 compartments. These classes also allow for differentiation and integration to evaluate the model's
 predictions.
+
 STRUCTURE:
+
 - Model(object)
 """
 
 
 class Model(object):
     """
     Helps create a model out of various disease compartments
+
     STRUCTURE:
+
         - __init__()
+        - get_deriv()
+        - integrate()
+        - add_layer()
     """
 
     def __init__(self, init_pop, layers=None, layer_names=None, layer_map=None):
         """
         Initialize the `Model` class
-
         All optional parameters can be added through the `add_layer` method
-        :param init_pop: initial population of the area in question
-        :param layers: =[], every layer in Model (as a class, e.g [Susceptible(), Infected(), Recovered()]
-        :param layer_names: =[], names of every previous layer in Model
-                                 (e.g [None, 'Susceptible', 'Infected'])
-                                allowed names are listed below:
-                                - Susceptible
-                                - Infected
-                                - Recovered
-                                - Exposed
-                                - Removed
-                                - Dead
-                                - Critical
-                                - Hospitalized
-                                use `None` to indicate that no other layer precedes the first layer.
-        :param layer_map: =[] next layers (as classes) for every layer in Model
-                              (e.g. [Infected(), Recovered(), None]) Use `None` to indicate that no other
-                              layer succeeds the last layer.
+
+        - init_pop: initial population of the area in question
+        - layers: =[], every layer in Model ( as a class, e.g [ Susceptible(), Infected(), Recovered() ] )
+        - layer_names: =[], names of every layer in Model (e.g ['Susceptible', 'Infected', 'Recovered'])\
+            allowed names are listed below:
+            - Susceptible
+            - Infected
+            - Recovered
+            - Exposed
+            - Removed
+            - Dead
+            - Critical
+            - Hospitalized
+        - layer_map: =[] next layers (as classes) for every layer in Model
+            (e.g. [Infected(), Recovered(), None])--use `[]` to indicate that no other

```

Comment:
If you put `[]` in \`\` then why don't you do this everywhere else? Such as on lines 32 and 31.

Reply:
Will add in future PR, for now we just need basic docs